source/manual/connect-to-vcloud-director.html.md

```diff
@@ -4,67 +4,78 @@ title: Connect to vCloud Director (Carrenza only)
 section: Infrastructure
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2019-08-08
+last_reviewed_on: 2019-11-26
 review_in: 6 months
 ---
 
-vCloud Director is the interface we use to manage our infrastructure in Carrenza.
-This includes virtual machines, gateways, firewalls and VPNs between providers.
+vCloud Director is the interface we use to manage our infrastructure in
+Carrenza. This includes virtual machines, gateways, firewalls and VPNs between
+providers.
 
 To access vCloud Director, you will need to connect to a Carrenza-provided VPN.
 You can use either Cisco AnyConnect or OpenConnect as a VPN client for this.
 
 ## Setting up the Cisco AnyConnect VPN profile on a Mac
 
-1. Clone the govuk-secrets private Git repo. The repo is very large but there is no need to download the full history.
+1. Clone the govuk-secrets private Git repo. The repo is very large but there
+   is no need to download the full history.
 
-    ```
-    mkdir -p ~/govuk && cd ~/govuk
-    git clone --depth 10 git@github.com:alphagov/govuk-secrets.git
+    ```sh
+    $ mkdir -p ~/govuk && cd ~/govuk
+    $ git clone --depth 10 git@github.com:alphagov/govuk-secrets.git
     ```
 
-1. Get the VPN client certificate and private key from the 2nd line [password store](https://github.com/alphagov/govuk-secrets/tree/master/pass) and save the decrypted contents to a file on your machine (for example ~/carrenza-vpn-cert-and-key.pem).
+1. Install `oathtool` (this will be used to generate one time passwords).
 
-    ```
-    cd govuk-secrets/pass
-    export PASSWORD_STORE_DIR=2ndline
-    pass carrenza/vpn-certificate > ~/carrenza-vpn-cert-and-key.pem
+   ```sh
+   $ brew install oath-toolkit
+   ```
+
+1. Get the VPN client certificate and private key from the 2nd line
+   [password store](https://github.com/alphagov/govuk-secrets/tree/master/pass)
+   and save the decrypted contents to a file on your machine (for example
+     `~/carrenza-vpn-cert-and-key.pem`).
+
+    ```sh
+    $ export PASSWORD_STORE_DIR=~/govuk/govuk-secrets/pass/2ndline
+    $ pass carrenza/vpn-certificate > ~/carrenza-vpn-cert-and-key.pem
```

Comment:
Make this one line?

Reply:
I could, however the `PASSWORD_STORE_DIR` environment variable is used later on, but I suppose we could combine both commands into one line.